calico.go

```diff
@@ -131,9 +165,9 @@ func cmdAdd(args *skel.CmdArgs) error {
 			// Don't create the veth or do any networking.
 			// Just update the profile on the endpoint. The profile will be created if needed during the
 			// profile processing step.
-			fmt.Fprintf(os.Stderr, "Calico CNI appending profile: %s\n", profileID)
+			log.Infof("Calico CNI appending profile: %s\n", profileID)
```

Comment:
shouldn't we be using logger here?

if there is a risk of accidentally using log rather than some predefined logger - IMHO I think it's a good idea to rename the log import to something other than log -- makes it less likely to make that mistake.